src/absil/ilread.fs

```diff
@@ -3943,7 +3943,8 @@ type ILReaderOptions =
       ilGlobals: ILGlobals
       reduceMemoryUsage: ReduceMemoryFlag
       metadataOnly: MetadataOnlyFlag
-      tryGetMetadataSnapshot: ILReaderTryGetMetadataSnapshot }
+      tryGetMetadataSnapshot: ILReaderTryGetMetadataSnapshot
+      isInteractive: bool }
```

Comment:
I think `isInteractive` should be called `requiresOnDiskAssemblies` or `requiresFilesFromRealFileSystem` or `bypassFileSystemShim` or something like that for most of the changes in this PR

Reply:
Should we introduce a different name in TcConfig/TcConfigBuilder in a separate PR as well? I could try coming up with a more descriptive one, like your examples.

Reply:
I think that could be tricky, there are several implications of the name that would be hard to convey